packages/backend/app/Services/Telegram.ts

```diff
@@ -0,0 +1,94 @@
+import axios from 'axios'
+import { DateTime } from 'luxon'
+
+import util from 'util'
+import { IMsgToTelegram } from './../../common/interfaces/IMsgToTelegram'
+import Logger from '@ioc:Adonis/Core/Logger'
+import { IPositiveResponse, INegativeResponse } from '../../common/interfaces/IResponse'
+import { ITelegram } from '../../common/interfaces/Itelegram'
+import telegramConfiguration from 'Config/telegram'
+import { creatingErrMsg, creatingOkMsg } from '../../common/helpers/creatingResponse'
+
+export default class TelegramBot {
+  private readonly config: ITelegram
```

Comment:
```suggestion
  private readonly config: ITelegramConfig
```
Или прямо telegramConfiguration как тип использовать.